health/job.py

```diff
@@ -83,6 +83,16 @@
 CONF = None
 
 
+def _get_driver(driver_type):
+    driver = None
+    try:
+        driver = importlib.import_module("." + driver_type + ".driver",
+                                         "health.drivers")
+    except ImportError:
+        logging.error("Could not load driver for '{}'".format(driver_type))
```

Comment:
raise

We should re-raise exception here and fall, it will be way simpler in real life to debug it than when you are hiding trace